produce_corrMatrix.py

```diff
@@ -0,0 +1,400 @@
+import ROOT
+import sys, os, pwd
+from subprocess import *
+import optparse, shlex, re
+import math
+import time
+from decimal import *
+import json
+from collections import OrderedDict as od
+from collections import defaultdict
+import seaborn as sns
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+
+#from plot_corr_func import *
+
+
+#sys.path.append('../inputs/')
+sys.path.append('datacardInputs/')
+
+from read_bins import *
+import yaml
+
+
+
+from higgs_xsbr_13TeV import *
+
+def parseOptions():
+
+    global opt, args, runAllSteps
+
+    usage = ('usage: %prog [options]\n'
+             + '%prog -h for help')
+    parser = optparse.OptionParser(usage)
+
+    # input options
+    parser.add_option('-d', '--dir',      dest='SOURCEDIR',type='string',default='./', help='run from the SOURCEDIR as working area, skip if SOURCEDIR is an empty string')
```

Comment:
This option is not used. So, why its here?

Please remove all options that we are not using.